test/test_matplotlib.py

```diff
@@ -1,3 +1,7 @@
+import os
+from selenium.webdriver.support.wait import WebDriverWait
```

Comment:
not that this file was moved to `packages/matplotlib/test_matplotlib.py` -- you will see it if you merge master in.

Reply:
Yes, I noticed, I guess we can do that once the whole `gsoc` branch gets merged with `master` i.e. we let them be separate OR should I actually move it inside the `packages/matplotlib` directory?

Reply:
Soon (doesn't have to be in this PR), we'll want to rebase the gsoc branch on the master branch to "shorten the distance" between the two.